osmoutils/cli_helpers.go

```diff
@@ -20,9 +20,9 @@ const (
 	bitlen = 64
 )
 
-func ParseUint64SliceFromString(s string, seperator string) ([]uint64, error) {
-	var parsedInts []uint64
-	for _, s := range strings.Split(s, seperator) {
+func ParseUint64SliceFromString(s string, separator string) ([]uint64, error) {
+	var parsedInts []uint64 //nolint:prealloc
```

Comment:
I think we should remove the prealloc linter, no need to add extra complexity to lots of our simple code

Reply:
we are of one mind